keyboards/zsa/voyager/info.json

```diff
@@ -0,0 +1,217 @@
+            {"matrix": [1, 1], "x": 0, "y": 21, "flags": 1},
+            {"matrix": [1, 2], "x": 17, "y": 21, "flags": 4},
+            {"matrix": [1, 3], "x": 34, "y": 19, "flags": 4},
+            {"matrix": [1, 4], "x": 52, "y": 17, "flags": 4},
+            {"matrix": [1, 5], "x": 69, "y": 19, "flags": 4},
+            {"matrix": [1, 6], "x": 86, "y": 21, "flags": 4},
+            {"matrix": [2, 1], "x": 0, "y": 32, "flags": 1},
+            {"matrix": [2, 2], "x": 17, "y": 32, "flags": 4},
+            {"matrix": [2, 3], "x": 34, "y": 30, "flags": 4},
+            {"matrix": [2, 4], "x": 52, "y": 28, "flags": 4},
+            {"matrix": [2, 5], "x": 69, "y": 30, "flags": 4},
+            {"matrix": [2, 6], "x": 86, "y": 32, "flags": 4},
+            {"matrix": [3, 1], "x": 0, "y": 43, "flags": 1},
+            {"matrix": [3, 2], "x": 17, "y": 43, "flags": 4},
+            {"matrix": [3, 3], "x": 34, "y": 41, "flags": 4},
+            {"matrix": [3, 4], "x": 52, "y": 39, "flags": 4},
+            {"matrix": [3, 5], "x": 69, "y": 41, "flags": 4},
+            {"matrix": [4, 4], "x": 86, "y": 43, "flags": 4},
+            {"matrix": [5, 0], "x": 86, "y": 53, "flags": 1},
+            {"matrix": [5, 1], "x": 96, "y": 58, "flags": 1},
+            {"matrix": [6, 0], "x": 138, "y": 10, "flags": 4},
+            {"matrix": [6, 1], "x": 155, "y": 10, "flags": 4},
+            {"matrix": [6, 2], "x": 172, "y": 8, "flags": 4},
+            {"matrix": [6, 3], "x": 190, "y": 5, "flags": 4},
+            {"matrix": [6, 4], "x": 207, "y": 8, "flags": 4},
+            {"matrix": [6, 5], "x": 224, "y": 10, "flags": 1},
+            {"matrix": [7, 0], "x": 138, "y": 21, "flags": 4},
+            {"matrix": [7, 1], "x": 155, "y": 21, "flags": 4},
+            {"matrix": [7, 2], "x": 172, "y": 19, "flags": 4},
+            {"matrix": [7, 3], "x": 190, "y": 17, "flags": 4},
+            {"matrix": [7, 4], "x": 207, "y": 19, "flags": 4},
+            {"matrix": [7, 5], "x": 224, "y": 21, "flags": 1},
+            {"matrix": [8, 0], "x": 138, "y": 32, "flags": 4},
+            {"matrix": [8, 1], "x": 155, "y": 32, "flags": 4},
+            {"matrix": [8, 2], "x": 172, "y": 30, "flags": 4},
+            {"matrix": [8, 3], "x": 190, "y": 28, "flags": 4},
+            {"matrix": [8, 4], "x": 207, "y": 30, "flags": 4},
+            {"matrix": [8, 5], "x": 224, "y": 32, "flags": 1},
+            {"matrix": [10, 2], "x": 138, "y": 43, "flags": 4},
+            {"matrix": [9, 1], "x": 155, "y": 43, "flags": 4},
+            {"matrix": [9, 2], "x": 172, "y": 41, "flags": 4},
+            {"matrix": [9, 3], "x": 190, "y": 39, "flags": 4},
+            {"matrix": [9, 4], "x": 207, "y": 41, "flags": 4},
+            {"matrix": [9, 5], "x": 224, "y": 43, "flags": 1},
+            {"matrix": [11, 5], "x": 128, "y": 58, "flags": 1},
+            {"matrix": [11, 6], "x": 138, "y": 53, "flags": 1}
+        ]
+    },
+    "tapping": {
+        "toggle": 1
+    },
```

Comment:
While on the PR under a different section `user preference configurable #defines should not be placed at the keyboard level` really should apply here too.

```suggestion
```